packages/forma-36-react-components/src/components/Tag/Tag.tsx

```diff
@@ -13,21 +14,29 @@ export type TagType =
 
 export const tagType = [
   'primary',
+  'filled-primary',
   'positive',
   'negative',
   'warning',
   'muted',
   'secondary',
 ];
 
-type Status = 'published' | 'draft' | 'archived' | 'changed' | 'deleted';
+type Status =
+  | 'published'
+  | 'draft'
+  | 'archived'
+  | 'changed'
+  | 'deleted'
+  | 'new';
 
 const statusTagTypeMap = {
   published: 'positive',
   draft: 'warning',
   archived: 'negative',
   changed: 'primary',
   deleted: 'negative',
+  new: 'filled-primary',
 };
```

Comment:
Wouldn't be easier to use an enum?

```ts
enum StatusTagType = {
  published = 'positive',
  ...
}

interface Props {
  status: StatusTagType // it means that status prop value needs to be of one of the enums keys (or value??)
}

console.log(StatusTagType.published) // it will log 'positive'
```

Reply:
yeah, it makes sense I think. 🤔  I will try to refactor it a bit.

Reply:
@gui-santos I think it don't make much differences tbh, I would keep it as it is or refactor on a separate branch. WDYT?